scripts/build_example

```diff
@@ -81,6 +85,23 @@ case "${language}" in
     exit 1;;
 esac
 
+if [[ "${docker_image}" == true ]]; then
+  if [[ "${language}" != "rust" ]]; then
+    echo "Docker images are only supported to Rust modules"
+    exit 1
+  fi
+
+  # Docker images only support the Asylo server.
+  "${SCRIPTS_DIR}/build_server" -s asylo
```

Comment:
Isn't this dependency implicit from the asylo_oak_docker Bazel rule?

Reply:
Deleted